CONTRIBUTING.md

```diff
@@ -46,10 +46,16 @@ Ready to contribute? Here's how to set yourself up for local development.
    $ git clone git@github.com:your_name_here/skfolio.git
    ```
 
-3. Install the project in development mode with the tests and linting dependencies:
+3. Install the project in development mode with the development dependencies:
 
+   Using **pip**:
    ```shell
-   $ pip install --editable ".[tests]"
+   $ pip install --editable ".[dev]"
+   ```
+
+   or using **uv**:
+   ```shell
+   $ uv pip install -r pyproject.toml --extra dev
```

Comment:
looks like also with `uv`, install should be in editable mode: `uv pip install -r pyproject.toml --editable . --extra dev`

Reply:
https://github.com/skfolio/skfolio/commit/a0b77a8220795e0636ccf2521c019bb364933e52